packages/kbn-ai-playground/server/routes.ts

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { fetchFields } from '../lib/fetch_query_source_fields';
+import Assist, { ConversationalChain, Prompt } from '@elastic/ai-assist';
+import { ChatOpenAI } from '@elastic/ai-assist/models';
+import { streamFactory } from '@kbn/ml-response-stream/server';
```

Comment:
Are we still planning to export `streamFactory` our of `@kbn/ml-response-stream/server` or use it as it is?

Reply:
I thought the plan was to check dependencies and if it has then will think about next steps, but as it doesn't we can just use this function